docs/source/api/v2/jobs.rst

```diff
@@ -31,23 +31,27 @@ Request Structure
+	+----------------------+----------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+	| cdn                  | no       | Return only invalidation jobs for delivery services with this CDN name                                                                                           |
+	+----------------------+----------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+	| createdBy            | no       | Return only invalidation jobs that were created by the user with this username                                                                                   |
+	+----------------------+----------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+	| deliveryService      | no       | Return only invalidation jobs that operate on the :term:`Delivery Service` with this :ref:`ds-xmlid`                                                             |
+	+----------------------+----------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+	| dsId                 | no       | Return only invalidation jobs pending on the :term:`Delivery Service` identified by this integral, unique identifier                                             |
+	+----------------------+----------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+	| id                   | no       | Return only the single invalidation job identified by this integral, unique identifer                                                                            |
+	+----------------------+----------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+	| keyword              | no       | Return only invalidation jobs that have this "keyword" - only "PURGE" should exist                                                                               |
+	+----------------------+----------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+	| maxRevalDurationDays | no       | Return only invalidation jobs with a start_time that is within the window defined by the ``maxRevalDurationDays`` :term:`Parameter` in :ref:`the-global-profile` |
```

Comment:
The field returned by the API is `startTime` - `start_time` is an internal DB column name with which consumers of the API need not concern themselves.